addons/l10n_ar/tests/test_manual.py

```diff
@@ -97,3 +98,17 @@ def test_14_corner_cases(self):
         """ Responsable Inscripto" in USD and VAT 21 """
         self._prepare_multicurrency_values()
         self._post(self.demo_invoices['test_invoice_10'])
+
+    def test_15_corner_cases(self):
+        """ RI partner with VAT exempt and 21. Test price unit digits """
+        self._post(self.demo_invoices['test_invoice_4'])
+        decimal_price_digits_setting = self.env.ref('product.decimal_price').digits
+        invoice_line_ids = self.demo_invoices['test_invoice_4'].invoice_line_ids
+        for line in invoice_line_ids:
+            l10n_ar_line_prices = line._l10n_ar_prices_and_taxes()
+            (unitary_part, l10n_ar_price_unit_decimal_part) = float_split_str(l10n_ar_line_prices['price_unit'], decimal_price_digits_setting)
```

Comment:
@pablohmontenegro 
to aviod ci/style error
```suggestion
            _unitary_part, l10n_ar_price_unit_decimal_part
 = float_split_str(l10n_ar_line_prices['price_unit'], decimal_price_digits_setting)
```